lib/ASRouter.jsm

```diff
@@ -511,10 +517,21 @@ class _ASRouter {
     }, {...DEFAULT_WHITELIST_HOSTS});
   }
 
+  _removePreviewMessage(state, lastMessageId) {
+    return {
+      messages: state.messages.filter(m => m.id !== lastMessageId.lastMessageId),
```

Comment:
is `lastMessageId` an object with a `lastMessageId` property? that seems weird.. can we re-name this to something less weird?